src/main/java/br/edu/ufcg/genus/services/SubjectService.java

```diff
@@ -166,6 +166,7 @@ private void checkAdminPermission(Subject subject, User user) {
         
         this.studentSubjectRepository.saveAll(studentSubjects);
         this.subjectRepository.saveAll(addedSubjects);
+				this.gradeService.saveGradeInRepository(grade);

```

Comment:
Remove these spaces behind the code "this.gradeService.saveGradeInRepository(grade);"

Reply:
Done.